src/main/java/hudson/plugins/accurev/ParseChangeLog.java

```diff
@@ -52,6 +49,13 @@ public static void setSubpath(String subpath) {
   public ChangeLogSet<AccurevTransaction> parse(
       Run build, RepositoryBrowser<?> browser, File changelogFile)
       throws IOException, SAXException {
+    try {
+      final EnvVars environment = build.getEnvironment(TaskListener.NULL);
+      subpath = environment.get("ACCUREV_SUBPATH");
```

Comment:
Why take the input via an env?
I assume this only works for a more recent version of AccuRev.
Otherwise the PR looks fine